File: src/uri/Accepted/GEOMETRY/P3047_A_idade_de_Dona_Monica.py
# Date: 03-09-2019
# author : Teerapat Phokhonwong
# Online judge: URI Online Judge
# Categories: BEGINNER
# Problem: 3047 - A idade de Dona Mônica
# Link: https://www.urionlinejudge.com.br/judge/en/problems/view/3047
# Timelimit: 1 sec
# Status: Accepted
# Submission: 4/2/20, 2:22:40 PM
# Language: Python3
# Runtime: 0.020s
# Solution:
# Note:

def max(a, b):
    if a > b:
        return a
    return b


if __name__ == "__main__":
    m = int(input())
    a = int(input())
    b = int(input())
    c = m - (a + b)
    brother = max(max(a, b), c)

    print(brother)

File: Java/URI_BEECROWD/BEGINNER/P3047_A_idade_de_Dona_Monica.java
/**
 * @author Teerapat Phokhonwong
 * @Onlinejudge: URI ONLINE JUDGE
 * @Categories: Beginner
 * @Problem: 3047 - A idade de Dona Mônica
 * @Link: https://www.urionlinejudge.com.br/judge/en/problems/view/3047
 * @Timelimit: 1
 * @Status: Accepted
 * @Submission: 4/20/20, 7:41:14 PM
 * @Runtime: 0.136s
 * @Solution:
 * @Note:
 */


package URI.Accepted.BEGINNER.sourcecode;

import java.util.Scanner;

public class P3047_A_idade_de_Dona_Monica {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int m = sc.nextInt();
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = m - (a + b);
        int brother = Math.max(Math.max(a, b), c);
        System.out.println(brother);
    }

}
